Update boosting docs to introduce manual boosting

Raindogjones opened the pull request.

Raindogjones committed:
Update boosting docs to reflect reintroduction

kecnry reviewed (commented):
Looks good to me, thanks!

Just a few small comments:
* can you rerun the output to avoid the error output not being able to connect to the passband server?
* For tutorials, I personally prefer to rewrite all dictionary twig-access using set/get methods instead with specific tags (`b.set_value(qualifier='sma', component='binary', value=...)`) to make it clear where parameters belong, to be efficient, and to try to stay consistent, but if you think this is easier to understand/maintain I won't insist.

Raindogjones committed:
Switched away from twigs

Raindogjones commented:
Sorry, I'm a twig man so wrote it as I would write all my scripts.  Hopefully more in line with the style of the rest of the docs now!

kecnry reviewed (approved):
Thanks!

Raindogjones committed:
New atmosphere docs

-Version bump in all docs (2.4->2.5)
-Update to atm_passbands to include new atmospheres
-New advanced atmosphere doc (atm_advanced) to explain ModelAtmosphere class and model atmosphere coverage

Raindogjones committed:
Update passbands.ipynb
